### PR TITLE
feature/KAS 4323 dossier VP

### PR DESCRIPTION
**Pull request by kobemertens**
https://kanselarij.atlassian.net/browse/KAS-4323

- chore: update SUBCASE_TYPES
- chore: remove unnecessary await
- chore: add send-to-vp-modal component
- chore: add vlaams parlement feature flag
- chore: add type properties to subcase
- chore: add send to VP button to actions
- chore: add translations
- chore: add document types to constants
- chore: add parliament related models
- chore: remove unneeded placeholder function
- chore: add correct piece type to warning message
- chore: add submitter to parliament-submission-activity
- add success toast on VP submit
- fix: some typos
- add comment
- add VP feature flag getter
- add parliament-flow to case model
- add alert when a parliament submission has been made
- add translations
- add (not functional) code to update UI

Related PRs:
- [ ] https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/429
- [ ] https://github.com/kanselarij-vlaanderen/vlaams-parlement-sync-service/pull/2

### CODE REVIEW

File: app/models/case.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ export default class CaseModel extends Model {
 
   @belongsTo('decisionmaking-flow', { inverse: 'case', async: true })
   decisionmakingFlow;
+  @belongsTo('parliament-flow', { async: true }) parliamentFlow;
```
Comment (tdn) [inline app/models/case.js]: making this inverse as mentioned in https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/429#pullrequestreview-1742804174

File: app/components/agenda/agendaitem/send-to-vp-modal.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+      }
+    }
+
+    return formattedMissingFiles;
+  };
+}
+
+function findPieceOfType(pieces, type, mimeType) {
+  return pieces.find((piece) => {
+    const documentContainer = piece.belongsTo('documentContainer').value();
```
Comment (tdn) [inline app/components/agenda/agendaitem/send-to-vp-modal.js:187]: I'm not a big fan of this way of loading async data. Will do for now, but should be fixed later

**Review by tdn — APPROVED** [review]
Code in `send-to-vp-modal` could be cleaner, but this works locally now.

File: app/components/agenda/agendaitem/send-to-vp-modal.js (hunk 2 above)
Comment (kobemertens) [reply app/components/agenda/agendaitem/send-to-vp-modal.js:187]: But this is not loading async data. Could you maybe elaborate on why you're not a big fan?

**Pull request merged**